clients/payment/src/components/CardPayment.tsx

```diff
@@ -253,7 +252,10 @@ export const CardPayment = ({ paymentLink, paymentMethodId, publicKey }: Props)
                     <View
                       style={[
                         styles.cardLogo,
-                        (isPaymentMethodValid === false || cardNumberState !== "valid") && {
+                        (isPaymentMethodValid === false ||
```

Comment:
You changed that for all types of card excepted `(isPaymentMethodValid === false || cardNumberState !== "valid")` that stays for `"cartes bancaires"`. Is this wanted?

Reply:
oupsy fixed